channel_output_webhook.py

```diff
@@ -16,22 +17,45 @@ class WebhookOutputChannel(OutputChannel):
 
 
     def do_send_alert(self, input_channel=None, canarydrop=None, **kwargs):
-        try:
-            payload = input_channel.format_webhook_canaryalert(
-                                          canarydrop=canarydrop,
-                                          **kwargs)
-            self.generic_webhook_send(simplejson.dumps(payload), canarydrop)
-        except Exception as e:
-            log.err(e)
+
+        slack = "https://hooks.slack.com"
+
+        if (slack in canarydrop['alert_webhook_url']):
+            try:
+                payload = input_channel.format_webhook_canaryalert(
+                                              canarydrop=canarydrop,
+                                              **kwargs)
+                self.generic_webhook_send(json.dumps(payload), canarydrop)
```

Comment:
We already using simplejson, and the only difference in this code path is that you use json (only difference here that I can see). Would you mind just using simplejson or elaborating why you want to use json instead? It would just make the code more compact.

Reply:
https://github.com/thinkst/canarytokens/pull/25#discussion_r240994927

tested this AM and pushed - https://github.com/thinkst/canarytokens/pull/25/commits/e71a10531bcadc51c7565b044c185332c23f45eb